### PR TITLE
Update codehs.user.css

### PR DESCRIPTION
**Pull request by s9176337**
talk was cheap

### CODE REVIEW

File: codehs.user.css [hunk 1]
```diff
@@ -527,10 +542,23 @@
...
+                        
+                        div.r:nth-child(even){
+                             background-color: @overlay1 !important;
+                        }
+                        
+                                                
+                        div.r:nth-child(odd){
+                             background-color: @overlay0 !important;
+                        }
+                    }
```
Comment (qDread-dev) [inline codehs.user.css:567]: Shouldnt the background be surface1 and surface0 here?

File: codehs.user.css [hunk 2]
```diff
@@ -527,10 +542,23 @@
             div.n {
                 // right bar
                 #right-tab-bar {
+                    
+                    
+                    .y {
+                        
+                        div.r:nth-child(even){
+                             background-color: @overlay1 !important;
```
Comment (qDread-dev) [inline codehs.user.css]:
```suggestion
                             background-color: @surface1 !important;
```

File: codehs.user.css [hunk 3]
```diff
@@ -527,10 +542,23 @@
...
+                        }
+                        
+                                                
+                        div.r:nth-child(odd){
+                             background-color: @overlay0 !important;
```
Comment (qDread-dev) [inline codehs.user.css]:
```suggestion
                             background-color: @surface0 !important;
```

File: codehs.user.css [hunk 4]
```diff
@@ -495,20 +510,30 @@
...
                                     .mtk8 {
-                                        color: @accent-color !important;
+                                        color: @overlay0 !important;
                                     }
                                     .mtk9 {
                                         color: @sapphire !important;
                                     }
+                                    // No this is string fr fr
+                                    .mtk20 {
+                                        color: @peach !important;
```
Comment (qDread-dev) [inline codehs.user.css]:
```suggestion
                                        color: @green !important;
```

**Review by qDread-dev — CHANGES_REQUESTED** [review]

File: codehs.user.css (hunk 1 above)
Comment (s9176337) [reply codehs.user.css:567]: 🤷

File: codehs.user.css (hunk 4 above)
Comment (s9176337) [reply codehs.user.css]: i thought strings were orange

Comment (qDread-dev) [reply codehs.user.css]: > i thought strings were orange

catppuccin has them as green

File: codehs.user.css [hunk 5]
```diff
@@ -1,10 +1,10 @@
 /* ==UserStyle==
 @name CodeHS Catppuccin
 @namespace https://github.com/qDread-dev/codehs-cat
-@version 0.0.27
+@version 0.0.28
 @updateURL https://raw.githubusercontent.com/qDread-dev/codehs-cat/refs/heads/main/codehs.user.css
 @description Soothing pastel theme for CodeHS
-@author qdread-dev
+@author qdread-dev TEEJIO
```
Comment (qDread-dev) [inline codehs.user.css]:
```suggestion
@author qdread-dev
```

userstyle header only supports 1 author
add credit in description?

File: codehs.user.css [hunk 6]
```diff
@@ -1,10 +1,10 @@
 /* ==UserStyle==
 @name CodeHS Catppuccin
 @namespace https://github.com/qDread-dev/codehs-cat
-@version 0.0.27
+@version 0.0.28
 @updateURL https://raw.githubusercontent.com/qDread-dev/codehs-cat/refs/heads/main/codehs.user.css
 @description Soothing pastel theme for CodeHS
```
Comment (qDread-dev) [inline codehs.user.css]:
```suggestion
@description Soothing pastel theme for CodeHS. By qdread-dev and TEEJIO
```

add credit in description

**Review by qDread-dev — CHANGES_REQUESTED** [review]

File: codehs.user.css (hunk 5 above)
Comment (s9176337) [reply codehs.user.css]: alr

**Review by qDread-dev — APPROVED** [review]
lgtm

**Pull request merged**